dev/tests/api-functional/testsuite/Magento/Sales/Service/V1/ShipmentGetTest.php

```diff
@@ -57,7 +58,14 @@ public function testShipmentGet()
         unset($data['tracks']);
         foreach ($data as $key => $value) {
             if (!empty($value)) {
-                $this->assertEquals($shipment->getData($key), $value, $key);
+                if($key === 'extension_attributes') {
+                    foreach ($value as $k => $val) {
```

Comment:
use more precise variable names instead of `$k` and `$val`
shortening is always bad practice